plaid/item.go

```diff
@@ -108,7 +108,32 @@ type createItemAddTokenRequest struct {
 }
 
 type ItemAddTokenUserFields struct {
-	ClientUserID string `json:"client_user_id"`
+	LegalName    string `json:"legal_name,omitempty"`
+	EmailAddress string `json:"email_address,omitempty"`
+	PhoneNumber  string `json:"phone_number,omitempty"`
+	// See FieldVerified
+	EmailAddressVerified *FieldVerified `json:"email_address_verified_time,omitempty"`
+	// See FieldVerified
+	PhoneNumberVerified *FieldVerified `json:"phone_number_verified_time,omitempty"`
```

Comment:
(nit) I think it can just be `FieldVerified` without the pointer since the struct itself contains a pointer

Reply:
That's deliberate - as per the comment on FieldVerified:

```
// FieldVerified indicates that this field has been verified if non-nil (e.g email
// address ownership verified by user acting on a verification email).
// Supply a time if the time the verification occurred is known
```